plugins/wasp/.mcp.json

```diff
@@ -0,0 +1,8 @@
+{
+  "mcpServers": {
+    "chrome-devtools": {
```

Comment:
What if they don't want to use it, but want instead to use Playwright?
Or, what if they already have chrome-devtools installed as mcp on their own?

I guess this will still get run and used by our plugin? Maybe that is fine for now, just wondering. Because maybe it also means we should not have this here in `.mcp.json` but instead it we should have it merely as a suggestion. Hm.